includes/header_footers.php

```diff
@@ -11,11 +11,12 @@ function printHeader($page_class_name, $scripts = [])
 		<!--[if IE]><link rel="shortcut icon" href="images/favicon.ico"><![endif]-->
 		<link rel="icon" href="images/favicon.png">
 		<meta name="author" content="Josh Greig">
-		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta name="viewport" content="width=device-width, user-scalable=no">
```

Comment:
I get that you can't scale with a native app but this change would also affect users of the website.  You want this specifically for when users are using it as a PWA, right?

Is there a way to determine if the application is running as a PWA and adjust the viewport content accordingly so PWA has user-scalable=no but everything else has initial-scale=1.0?

Reply:
I honestly just copied this from a tutorial I was reading, it may not be necessary.

I'll try to remove.

Reply:
Actually, I looked it up. We probably want "user-scale=no" because it prevents the user from being able to pinch to zoom on mobile devices.

and just want to note that it does not stop a desktop browser from being able to zoom.